test_script_for_running_modified_main_loop.py

```diff
@@ -0,0 +1,4 @@
+import sys
+sys.path.append("/home/pi/Github_repo/preference_learning_dephy_exo")
```

Comment:
nit: maybe add an instruction this should be changed based on persons' file structure?